Lib/test/test_os.py

```diff
@@ -101,6 +95,10 @@ def create_file(filename, content=b'content'):
                                        'on AIX, splice() only accepts sockets')
 
 
+def tearDownModule():
+    asyncio.set_event_loop_policy(None)
```

Comment:
The line that set a loop policy is not present any more, so this cleanup step doesn’t seem to apply anymore.

Reply:
https://github.com/python/cpython/runs/5604663664?check_suite_focus=true

```
901: == Tests result: ENV CHANGED ==

```

Reply:
If I understand [a failed pre-policy run](https://github.com/python/cpython/runs/5604663664?check_suite_focus=true) correctly:

```plain
0:04:29 load avg: 3.56 [300/433/1] test_os failed (env changed) -- running: test_asyncio (1 min 10 sec), test_concurrent_futures (1 min 57 sec)

[a bunch of oks and skips follow]

[...]

423 tests OK.

10 slowest tests:
- test_tools: 4 min 44 sec
- test_concurrent_futures: 3 min 30 sec
- test_peg_generator: 2 min 57 sec
- test_multiprocessing_spawn: 2 min 27 sec
- test_gdb: 1 min 51 sec
- test_asyncio: 1 min 50 sec
- test_multiprocessing_forkserver: 1 min 38 sec
- test_multiprocessing_fork: 1 min 22 sec
- test_regrtest: 1 min 12 sec
- test_statistics: 49.1 sec

1 test altered the execution environment:
    test_os

9 tests skipped:
    test_devpoll test_ioctl test_kqueue test_msilib test_startfile
    test_winconsoleio test_winreg test_winsound test_zipfile64

Total duration: 11 min 50 sec
Tests result: ENV CHANGED
make: *** [Makefile:1795: buildbottest] Error 3
Error: Process completed with exit code 2.
```

the policy must either be manually set to something before any case is runned or be returned back to `None` before the suit exits. I've chosen the first variant, Andrew proposed to switch to the second.